### PR TITLE
Creates CONTRIBUTING.md

### LINKED ISSUE #171: Add Contributing.md
Part of #51

### PR DESCRIPTION
**Pull request by tessaSAC**
fixes #171

### CODE REVIEW

**Comment by tessaSAC** [comment]
Also addresses part of #51

File: CONTRIBUTING [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+-----
+
+
+## Before You Start
+
+
+### Code of Conduct
+
+This project adheres to the Contributor Covenant [code of conduct](CODE_OF_CONDUCT.md).
```
Comment (prestonvanloon) [inline CONTRIBUTING]: Can you copy a code of conduct into this PR?

File: CONTRIBUTING [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+------
+
+
+## On Style
+
+
+### Git Commit Messages
+
+* Use the present tense
```
Comment (kirjs) [inline CONTRIBUTING]: examples?

File: CONTRIBUTING (hunk 1 above)
Comment (tessaSAC) [inline CONTRIBUTING]: Oops, bad link! Sorry 🙏

Do you mean link to one or paste it directly into the file? Is there one we prefer (if not this one)/should it be written from scratch?

**Pull request merged**